package.json

```diff
@@ -44,10 +44,15 @@
       "prettier --write"
     ]
   },
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "not ie <= 10"
+  ],
   "dependencies": {
-    "@ant-design/icons": "^4.2.1",
-    "@ant-design/pro-layout": "6.0.0-2",
-    "@ant-design/pro-table": "^2.3.3",
+    "@ant-design/icons": "^4.0.0",
```

Comment:
I'm not sure if we should use some older versions...

Reply:
`^` Should mean the maximum version greater than 4.0.0.

In [yarn.lock](https://github.com/apache/incubator-apisix-dashboard/blob/af555bc839759dca1b5268ecf523e789436a1fbc/yarn.lock#L30) can prove it.